### PR TITLE
fix driver classpath issue in init_spark_on_local

### PR DESCRIPTION
**Pull request by jenniew**
Change drive classpath from sparkconf to submit option.

Related issue: https://github.com/intel-analytics/analytics-zoo/issues/2522

### CODE REVIEW

**Comment by hkvision** [comment]
In this way, only `init_spark_on_local` works but `init_nncontext` doesn't work.

**Comment by jenniew** [comment]
> In this way, only `init_spark_on_local` works but `init_nncontext` doesn't work.

Move set submit opts to init_spark_conf

File: pyzoo/zoo/util/spark.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import os
 import glob
 
-from pyspark import SparkContext
+from bigdl.util.common import *
```
Comment (hkvision) [inline pyzoo/zoo/util/spark.py:20]: Then we don't need to change this file.

**Comment by hkvision** [comment]
Need to manually test on server before merging this.

**Comment by jenniew** [comment]
> Need to manually test on server before merging this.

I already tested it on almaren with init_nncontext and init_spark_on_local.

File: pyzoo/zoo/util/spark.py (hunk 1 above)
Comment (jenniew) [reply pyzoo/zoo/util/spark.py]: updated

**Comment by jenniew** [comment]
Jenkins pass: http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/3713/

**Pull request merged**